extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -150,7 +150,11 @@ protected function saveToParserOutput( \ParserOutput $parserOutput, Nodes\NodeIn
 		// (see: PortableInfoboxDataService.class.php)
 		if ( $raw ) {
 			$infoboxes = json_decode( $parserOutput->getProperty( PortableInfoboxDataService::INFOBOXES_PROPERTY_NAME ), true );
-			$infoboxes[] = [ 'data' => $raw->getRenderData(), 'sourcelabels' => $raw->getSourceLabel() ];
+			$infoboxes[] = [
+				'data' => $raw->getRenderData(),
+				'sourcelabels' => $raw->getSourceLabel(),
+				'sourcetypes' => $raw->getSourceType()
```

Comment:
This name confuses me. I would think that this will get you a list of sources and their types, where in fact it provides a list of sources and the associated row type.